lattice-control/lattice-control-interface.smithy

```diff
@@ -124,12 +124,13 @@ operation StopProvider {
 /// Requests that an actor be stopped on the given host
 operation StopActor {
     input: StopActorCommand
-    output: CtlOperationAck
+    output: CtlOperationAck    
```

Comment:
Super nit: I keep finding these extra spaces everywhere and I don't know what is putting them there

Reply:
I think vscode does it :(